.rubocop.yml

```diff
@@ -0,0 +1 @@
+inherit_from: .rubocop_todo.yml
```

Comment:
Do we need any of this stuff here? https://github.com/sul-dlss/argo/blob/master/.rubocop.yml#L2-L13

Reply:
We don't have any of those paths... Set the version though.